paseos/attitude/geometric_model.py

```diff
@@ -0,0 +1,123 @@
+from loguru import logger
+import numpy as np
+import trimesh
```

Comment:
`trimesh` needs to be added to if we want to use it 
https://github.com/aidotse/PASEOS/blob/e5bb071b4ee2988c2765497aa8fe3d0f6eacb655/environment.yml#L4
and
https://github.com/aidotse/PASEOS/blob/e5bb071b4ee2988c2765497aa8fe3d0f6eacb655/setup.py#L14
and
https://github.com/aidotse/PASEOS/blob/e5bb071b4ee2988c2765497aa8fe3d0f6eacb655/requirements.txt#L1

(sorry, pip & conda have to many divergent wishes 😢 )

Reply:
Added trimesh>=4.0.7 to all three files (version of Trimesh I currently have)